scripts/g.extension/g.extension.py

```diff
@@ -139,6 +143,8 @@
 #% required: extension, -l, -c, -g, -a, -j
 #% exclusive: extension, -l, -c, -g
 #% exclusive: extension, -l, -c, -a
+#% requires: -o, url
+#% requires: branch, url
```

Comment:
It is not necessary to define the relationship between the `url` and the `branch` option. Branch option has default answer `main`.